### PR TITLE
LL-1455 Improved ui for camera error on qrcode scan

### PR DESCRIPTION
**Pull request by juan-cortes**
Screenshot 2019-07-04 at 10 37 24

### Type

UI Polish

### Context

https://ledgerhq.atlassian.net/browse/LL-1455

### Parts of the app affected / Test plan

Send flow, qrcode scan

### CODE REVIEW

File: static/i18n/en/app.json [hunk 1]
```diff
@@ -878,6 +879,10 @@
       "title": "That's the wrong app",
       "description": "Open the ‘{{managerAppName}}’ app on your device"
     },
+    "NoAccessToCamera": {
+      "title": "",
```
Comment (gre) [inline static/i18n/en/app.json]: if there is no text, we shouldn't put that in wordings to translate.

File: static/i18n/en/app.json [hunk 2]
```diff
@@ -878,6 +879,10 @@
...
+      "description": "Please ensure your system has a camera, that it's not in use by another application and that Ledger Live has permission to use it."
```
Comment (gre) [inline static/i18n/en/app.json]: @dasilvarosa please confirm. thanks!

Comment (gre) [reply static/i18n/en/app.json]: @juan-cortes  I feel. we need to use title. not description. We have a guarantee in errors that a title is always set, only description is optional.

File: src/components/QRCodeCameraPickerCanvas.js [hunk 3]
```diff
@@ -152,10 +213,10 @@ export default class QRCodeCameraPickerCanvas extends Component<
             this.unsubscribes.push(() => cancelAnimationFrame(raf))
           }
         })
-        .catch(e => {
+        .catch(_ => {
           if (this.unmounted) return
           this.setState({
-            message: String(e.message || e.name || e),
+            message: <Trans i18nKey="errors.NoAccessToCamera.description" />,
```
Comment (gre) [inline src/components/QRCodeCameraPickerCanvas.js]: @juan-cortes I would prefer we have a new `NoAccessToCamera` error, we `setState({ error: new NoAccessToCamera(e.message) })` and we use `<TranslatedError` in the rendering part. it's the general convention we use everywhere.

File: static/i18n/en/app.json (hunk 2 above)
Comment (juan-cortes) [reply static/i18n/en/app.json]: The wording comes directly from @dasilvarosa , so very little trust..

File: src/components/QRCodeCameraPickerCanvas.js (hunk 3 above)
Comment (juan-cortes) [reply src/components/QRCodeCameraPickerCanvas.js]: Sure.... I'll make a pr on common to add the error

**Comment by juan-cortes** [comment]
Needs https://github.com/LedgerHQ/ledgerjs/pull/376/files and errors bumped now.

**Comment by gre** [comment]
live-common@7.1.0 + ledgerjs bumps

**Comment by juan-cortes** [comment]
7.2.1 :trollface:  @gre

**Comment by dasilvarosa** [comment]
I am DS and I approve of @juan-cortes 's copy-pasting skills.

**Comment by juan-cortes** [comment]
Thank you @dasilvarosa !
We only need @Arnaud97234 test now

**Comment by gre** [comment]
we'll test on develop.

**Pull request merged**